# Spatial Transcriptomics Reveals Expression Gradients in Developing Wheat Inflorescences at Cellular Resolution </p>
</p>
</p>
<i>Scripts supporting publication Long Lister et al., 2024</i> </p>
<i>Updated for version 2 of manuscript, for historical scripts see 'version 1' folder</i> </p>
</p>
</p>
<b>cellpose2_cyto2_segmentationspecification.json</b> </p>
Segmentation specifications used with Cellpose2 </p>
</p>
</p>
<b>Impulse_Central_vs_Basal.ipynb</b> </p>
ImpulseDE2 script generating differential expression dataset from bulk RNA-seq</p>
</p>
</p>
<b>QC_Filtering.ipynb</b> </p>
This scripts details the import of spatial transcriptomic data, normalisation and filtering, generating QC information and figure generation</p>
</p>
</p>
<b>SampleIntegrations_Clustering_GeneEnrichment.ipynb</b> </p>
This scripts details sample integration, leiden clustering, and gene enrichment analysis of spatial transcriptomic datasets </p>
</p>
</p>
<b>VRT_SEP_expression_analysis.ipynb</b> </p>
This script details the analysis of VRT2 and SEP1-4 expression patterns in wheat inflorescence tissue. Scripts detail the generation of transverse bins & gene expression gradient quantification, and well as gene co-localisation</p>
</p>
</p>
<b>SpikeletRidge_LeafRidge_analysis.ipynb</b> </p>
Spikelet ridge & leaf ridge gene expression analysis of late double ridge spikes 
</p>
